infra/build/functions/build_lib.py

```diff
@@ -94,6 +96,26 @@ def get_targets_list_url(bucket, project, sanitizer):
   return url
 
 
+def dockerify_run_step(step, use_arm, armify_image_name=False):
```

Comment:
Could we pass `architecture` instead of `use_arm`? 

And also for `armify_image_name`, call it `platform_specific_image`.

Reply:
Done.